src/Microsoft.ML.Maml/Microsoft.ML.Maml.csproj

```diff
@@ -4,7 +4,10 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <DefineConstants>CORECLR</DefineConstants>
     <IncludeInPackage>Microsoft.ML</IncludeInPackage>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
```

Comment:
Hi @jwood803 thanks for looking at this. The current separation is that we have this library which is a standard library (and must remain one, since other standard projects have it as a dependency), as well as the project "Console" that builds the `MML.dll` and *that* can be run via `dotnet mml.dll`. So I might have expected that project `Microsoft.ML.Console` to be bundled as a tool, as opposed to this library, which again must be standard and so cannot be made to be core. Does it make sense? Is there a a barrier to doing it?

Reply:
That is my understanding as well. The actual executable is located at: https://github.com/dotnet/machinelearning/blob/master/src/Microsoft.ML.Console/Microsoft.ML.Console.csproj.  While this `Maml` project is a library that contains all the logic in it.